pages/login.js

```diff
@@ -0,0 +1,90 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { Box, Flex, Heading, Text, Link } from 'ooni-components'
+import { useRouter } from 'next/router'
+import NLink from 'next/link'
+import Head from 'next/head'
+
+import NavBar from 'components/NavBar'
+import Layout from 'components/Layout'
+import LoginForm from 'components/login/LoginForm'
+import { loginUser } from '/lib/api'
+import { mutate } from 'swr'
+import SpinLoader from 'components/vendor/SpinLoader'
+import useUser from 'hooks/useUser'
+import { FormattedMessage } from 'react-intl'
+
+const Login = () => {
+  const router = useRouter()
+  const { token } = router.query
+
+  const redirectTo = typeof window !== 'undefined' && window.location.origin
+
+  const { user, loading, submitted, setSubmitted, loggedIn, reqError } = useUser()
+
+  // // If user is already logged in, redirect to home page
+  // useEffect(() => {
+  //   // if (!loading && user.loggedIn) {
+  //   //   router.replace('/')
+  //   // }
+  // }, [user, loading, router])
+
+  return (
+    <Layout title='Login'>
+      <Head>
+        <title>Login</title>
```

Comment:
Maybe for the login page we should inherit the title tags coming from the rest of OONI Explorer and mention somewhere in here that it's a login relative to OONI Explorer.